src/components/graph/Graph.jsx

```diff
@@ -424,6 +440,35 @@ export default class Graph extends React.Component {
         }
     };
 
+    /**
+     * Obtain a set of properties which will be used to perform the focus and zoom animation if
+     * required. In case there's not a focus and zoom animation in progress, it should reset the
+     * transition duration to zero and clear transformation styles.
+     * @returns {Object} - Focus and zoom animation properties.
+     */
+    _generateFocusAnimationProps = () => {
+        const { focusedNodeId } = this.props.data;
```

Comment:
Why not reading this from `this.state` instead?

Reply:
Until now, the `focusedNodeId` wasn't been stored in the component state, so the only way to retrieve that data was by reading the props. 

I uploaded a new commit by creating a new `focusedNodeId` property in the state, so it's possible to get it by doing `this.state` instead.